libraries/Wire/src/Wire.cpp

```diff
@@ -354,7 +354,7 @@ uint8_t TwoWire::endTransmission(bool sendStop)
     return 4;
 }
 
-uint8_t TwoWire::requestFrom(uint16_t address, uint8_t size, bool sendStop)
+size_t TwoWire::requestFrom(uint16_t address, size_t size, bool sendStop)
```

Comment:
Are we OK doing this from a backwards compatibility perspective? Will folks get compiler warnings/errors if they pass in a `uint8_t` rather than a `size_t`? Or if they try and assign the `size_t` to an existing `uint8_t` variable?

Reply:
Also, the header file ought to be updated.

Reply:
So I don't actually change the full header definition sets, If you check closely, I swapped two of them ([1](https://github.com/espressif/arduino-esp32/pull/5935/files#diff-90b03e94086217edc0bf5abd688554e4165b4d7401b8413dc18c27e3263378d8R357) & [2](https://github.com/espressif/arduino-esp32/pull/5935/files#diff-90b03e94086217edc0bf5abd688554e4165b4d7401b8413dc18c27e3263378d8R462) )so it will come out to the exact same thing in the end.

Reply:
Wait, just noticed the difference in address (uint16_t vs uint8_t) so I guess I will need to modify the headers